### DISCUSSION TITLE
Deployment models / Multicluster Community discussions

### DISCUSSION THREAD

**Discussion by lucasponce** [kiali/kiali#4013] [Multicluster]
Hello all,

As you can see in the discussions & issues, Istio and Kiali multicluster features are in active development and there are several discussions around priorities about which options to support first [1] in Kiali.

We are interested to collect your feedback to drive these efforts in Kiali, so in the next Sprint 57 review planned for May 27th we'd discuss with the community potential options and roadmap.

I'm inviting anyone interested in the topic and explictly users like @feuyeux @LeslieKuo @rwong2888 @ethernoy that have opened issues and discussions around the topic (apologizes, probably I missed more users).

Please, let us know if that meeting would be interesting for you, if you could attend and if you have any preference for time zone, typically we setup a meeting between Europe and East Coast time, but depending of the attendees we can find other alternatives.

Thanks in advance !

[1] https://istio.io/latest/docs/ops/deployment/deployment-models/

**Comment by ethernoy** [comment]
Hi @lucasponce, thank you so much for inviting me and I am very interested to attend. I live in UTC+8 timezone, so I am not available from 00:00 UTC+8 to 09:00 UTC+8.

**Comment by rwong2888** [comment]
I'd like to attend. I am on the east coast. Please let us know the details.

**Comment by LeslieKuo** [comment]
Hi @lucasponce, Big thanks for your inviting. I am very glad to join the meeting. I am also in UTC-8 time zone. So I am not available from 00:00 UTC+8 to 09:00 UTC+8. If there is any updates about the meeting, please let me know. Always check mail  676365950@qq.com. Thanks a million

**Comment by lucasponce** [comment]
Thanks for the responses.
We're going to promote this meeting during the week to collect feedback from users and at the end of the week we'll create one/two sessions that may be time friendly for a Asia + Europe/USA timezones.

**Comment by feuyeux** [comment]
cool, thanks lucas. And I'm appreciate to join this discuss, my zone is UTC+8, too.

**Comment by lucasponce** [comment]
Hi all, thanks for the feedback ! 

We will have two meetings, let us know if it works.

A first meeting in CEST + APAC time friendly:

https://bluejeans.com/7625852117
Thursday May 27th, 2021
[CEST] 10.00am - 10.30am
[UTC+8] 04.00pm - 04.30pm

A second meeting where we will have our regular Sprint Summary+Demo + Community meeting:

https://bluejeans.com/7625852117
Thursday May 27th, 2021
[CEST] 04.00pm  - 04.30pm
[EDT] 10.00am - 10.30am

Note that we can set up another meeting later if these two doesn't work for UTC-8 folks (cc @LeslieKuo let us know).

This will be updated on the twitter account https://twitter.com/KialiProject so, feel free to follow for notifications or news.

The goal of the meeting is to discuss with the community the potential options to support in Kiali about the deployment models [1], so, your feedback will help us to define priorities and needs !

Thanks !

[1] https://istio.io/latest/docs/ops/deployment/deployment-models/

**Comment by rwong2888** [comment]
The first meeting would be at 4 am Eastern for me, so would not be able to join, but I can do the 2nd meeting.

**Reply by lucasponce** [reply]
Yes, first meeting would be for Asia + Europe folks and second one for America + Europe.
And in any case, if we need a third one for America Pacifig time, let us know.

Thanks !

**Comment by LeslieKuo** [comment]
Roger it. Able to join both of them. Thank you so much.

**Comment by lucasponce** [comment]
Some notes from the feedback collected from my side:
- Both Multi-Primary and Primary-Remote models are used.
- Multi-Primary scenarios may be used in meshes with few cluster (<5).
- Primary-Remote scenarios may be used for greater clusters, where Primary cluster can act as an "administration cluster".
- As discussed in https://github.com/kiali/kiali/discussions/4013 Primary-Remote may involve more combinations where Kiali can be deployed on the Primary and has write rights on that cluster, but only view only rights to the other clusters.

Thanks a lot for the APAC discussion ! @feuyeux @ethernoy @LeslieKuo (sorry, I guess I missed one github user) and let me know if I missed anything !

**Reply by abonas** [reply]
A few more notes from APAC discussion, please correct me if something was not documented correctly.
- Primary cluster with remote cluster - control plane is only on primary cluster
- Multi/primary - Istio is in both clusters
- "control plane cluster" - IstioD is in one cluster, where there are no deployments from customers, and no Kiali. Kiali + customer deployments are installed in second cluster. P8s is installed where Kiali is.
